### PR TITLE
timing CR for content archives

### PR DESCRIPTION
**Pull request by Atlante45**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7528/app_12503_PR.apk

File: domain-server/src/AssetsBackupHandler.cpp [hunk 1]
```diff
@@ -24,12 +24,13 @@
...
-static const QString ASSETS_DIR = "/assets/";
-static const QString MAPPINGS_FILE = "mappings.json";
-static const QString ZIP_ASSETS_FOLDER = "files";
-
 using namespace std;
 
+static const QString ASSETS_DIR { "/assets/" };
+static const QString MAPPINGS_FILE { "mappings.json" };
+static const QString ZIP_ASSETS_FOLDER { "files" };
+static const chrono::minutes MAX_REFRESH_TIME { 15 };
```
Comment (birarda) [inline domain-server/src/AssetsBackupHandler.cpp]: This seems like a really high max to me

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7528/HighFidelity-Beta-PR12503-3592fcf3c2d89b893dd77737ac3e618d5632d273.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7528/HighFidelity-Beta-PR12503-3592fcf3c2d89b893dd77737ac3e618d5632d273.dmg

File: domain-server/src/AssetsBackupHandler.cpp (hunk 1 above)
Comment (Atlante45) [reply domain-server/src/AssetsBackupHandler.cpp]: Any suggestion what a good value would be, 5 minutes?
(Refreshes occur every 30 sec or so. Varies depending on how fast the AS responds and on the coarse timer.

Comment (birarda) [reply domain-server/src/AssetsBackupHandler.cpp]: 5 minutes seems more reasonable to me

Comment (Atlante45) [reply domain-server/src/AssetsBackupHandler.cpp]: fixed

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7533/app_12503_PR.apk

**Pull request merged**